app/e2e/storageState.json

```diff
@@ -0,0 +1,35 @@
+{
+  "cookies": [
+    {
+      "sameSite": "Lax",
+      "name": "_ga_CJ6VQRJ3TC",
+      "value": "GS1.1.1657245813.1.0.1657245813.0",
+      "domain": "localhost",
+      "path": "/",
+      "expires": 1720317813,
+      "httpOnly": false,
+      "secure": false
+    },
+    {
+      "sameSite": "Lax",
+      "name": "_ga",
+      "value": "GA1.1.1342012968.1657245814",
+      "domain": "localhost",
+      "path": "/",
+      "expires": 1720317813,
+      "httpOnly": false,
+      "secure": false
+    },
+    {
+      "sameSite": "Lax",
+      "name": "jwt",
+      "value": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IlZYTmxjam94TVdRM1l6VmxNUzB6T1RBM0xUUmpNR0V0T0dOaE15MDBabVZrTXpWaU16QTVNbU09IiwiaWF0IjoxNjU3MjQ1ODE1fQ.a1vvv9h3hFR90-IuTBX5itbQUckX-4VCXInUM9qlS98",
+      "domain": "localhost",
+      "path": "/",
+      "expires": -1,
+      "httpOnly": false,
+      "secure": false
+    }
+  ],
+  "origins": []
```

Comment:
Does this have to be .json or could it be .js with module.exports? I think it would be helpful to add comments to the values. Otherwise, it seems just like magic.

Reply:
This is an auto-generated file from the test-setup.